Python/specialize.c

```diff
@@ -637,3 +640,103 @@ _Py_Specialize_LoadGlobal(
     cache0->counter = saturating_start();
     return 0;
 }
+
+/* TODO:
+    - Specialize calling C types like int() with CALL_CTYPE
+    - Specialize python function calls.
+*/
+int
+_Py_Specialize_CallFunction(PyObject *builtins,
```

Comment:
Remove the `builtins` parameter. It is only used for stats which will be mostly turned off.
We could `#ifdef` the extra parameter, but I don't think it is worth the extra complexity.